From c6f37e778953f8e83276541091074bde038a043c Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Mon, 7 Mar 2016 07:45:44 +0000
Subject: [PATCH 2/2] xfsettingds: Use the correct (stateless) desktop file
 with SmClient

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 xfsettingsd/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xfsettingsd/main.c b/xfsettingsd/main.c
index a3a14fe..af8bad9 100644
--- a/xfsettingsd/main.c
+++ b/xfsettingsd/main.c
@@ -67,7 +67,7 @@
 #endif
 
 #define XFSETTINGS_DBUS_NAME    "org.xfce.SettingsDaemon"
-#define XFSETTINGS_DESKTOP_FILE (SYSCONFIGDIR "/xdg/autostart/xfsettingsd.desktop")
+#define XFSETTINGS_DESKTOP_FILE (DATADIR "/xdg/autostart/xfsettingsd.desktop")
 
 #define UNREF_GOBJECT(obj) \
     if (obj) \
-- 
2.19.1

